### PR TITLE
Add stand-alone jenkinsfile to trigger external and jck parents job directly

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
jenkins job https://ci.adoptopenjdk.net/view/work-in-progress/job/ExternalAndJckBuilds/

File: buildenv/jenkins/parentsJobPipeline.groovy [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!groovy
+def javaVersions = params.javaVersions.trim().split("\\s*,\\s*");
+def buildLists = params.buildLists.trim().split("\\s*,\\s*");
```
Comment (llxia) [inline buildenv/jenkins/parentsJobPipeline.groovy:3]: Should we use the same parameter names as testJobTemplate? i.e., javaVersions -> JDK_VERSIONS, buildLists -> GROUPS

Comment (sophia-guo) [reply buildenv/jenkins/parentsJobPipeline.groovy:3]: Yes, definitely. The reason I'm using the javaVersions is that will be consistent with upstream naming convention as the configuration files are from upstreams.

Comment (sophia-guo) [reply buildenv/jenkins/parentsJobPipeline.groovy]: Updated.

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]

**Pull request merged**